Loop/Models/ManualGlucoseState.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ManualGlucoseState.swift
+//  Loop
+//
+//  Created by Nathaniel Hamming on 2020-09-18.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import LoopKit
+
+struct ManualGlucoseState: SensorDisplayable {
```

Comment:
ManualGlucoseState -> ManualGlucoseDisplayable?  And SensorDisplayable -> GlucoseDisplayable?  The latter is LoopKit, I know, and maybe we do it later, but if this protocol is used to display both sensor and manual glucose data, the name could use updating.

Reply:
I'll definitely update `GlucoseDisplayable` now. I was confused when writing the code and happy to take this suggestion.